static/app/views/issueList/searchBar.tsx

```diff
@@ -1,4 +1,6 @@
 import {useCallback} from 'react';
+// eslint-disable-next-line no-restricted-imports
+import {withRouter, WithRouterProps} from 'react-router';
```

Comment:
Don't like using this instead of `useLocation`, but the hook broke over 30 tests 😞

Reply:
ah I see yeah